js/stat_test.ts

```diff
@@ -54,7 +54,7 @@ testPerm({ read: true }, async function lstatSyncSuccess(): Promise<void> {
   assert(!modulesInfo.isDirectory());
   assert(modulesInfo.isSymlink());
 
-  const testsInfo = Deno.lstatSync("tests");
+  const testsInfo = Deno.lstatSync("website");
```

Comment:
Maybe update the variable name too?
As far as I'm concerned it could be just `const info2 = ...`.
But **tests**Info = Deno.lstatSync("**website**") is kinda weird.

Reply:
done